sdk/include/reactos/shellutils.h

```diff
@@ -627,6 +633,19 @@ static inline PCUIDLIST_RELATIVE HIDA_GetPIDLItem(CIDA const* pida, SIZE_T i)
 
 #ifdef __cplusplus
 
+#if defined(CMIC_MASK_UNICODE) && defined(SEE_MASK_UNICODE)
+static inline bool IsUnicode(const CMINVOKECOMMANDINFOEX &ici)
```

Comment:
LPCMINVOKECOMMANDINFO can't be NULL and if you want a NULL check and return E_INVALIDARG, that needs to happen before the Unicode check anyway so a reference is fine.

Reply:
ah no, I wasn't expecting for doing NULL checks (if pointer and annotations were used, it would be `_In_ LP...`, so if a NULL was given, a crash would ensue, no problem).